client/marklogicClient.ts

```diff
@@ -358,24 +370,26 @@ export async function filterJsServerByConnectedStatus(
     const filteredChoices: MlxprsQuickPickItem[] = [];
     choices.forEach(async (choice) => {
         const url = ClientContext.buildUrl(dbClientContext.params.host, `/jsdbg/v1/connected/${choice.label}`, dbClientContext.params.ssl, managePort);
-        const options = {
+        const headers = {
             headers: {
                 'Content-type': 'application/x-www-form-urlencoded',
                 'X-Error-Accept': 'application/json'
-            },
-            auth: {
-                user: dbClientContext.params.user,
-                pass: dbClientContext.params.pwd,
-                'sendImmediately': false
             }
         };
+        const options = {
+            headers,
+            username: dbClientContext.params.user,
```

Comment:
Just confirming, mlxprs only supports digest/basic auth today, right? This of course won't work with MarkLogic Cloud or any other non-username/password auth strategy that MarkLogic supports. I think that's fine for the 3.x timeframe as it's likely always been the case for mlxprs. But at some point, we'd want mlxprs to support every auth strategy. 

That makes me wonder - instead of trying to upgrade `request` to a separate HTTP client - which ultimately requires supporting every auth type possible - is the better longterm approach here to figure out how to provide a backdoor in the Node Client for invoking any URL in MarkLogic? That would also need to include reusing the recently-added `basePath` support.